src/raftstore/coprocessor/properties.rs

```diff
@@ -24,22 +24,6 @@ const PROP_NUM_VERSIONS: &str = "tikv.num_versions";
 const PROP_MAX_ROW_VERSIONS: &str = "tikv.max_row_versions";
 const PROP_ROWS_INDEX: &str = "tikv.rows_index";
 const PROP_ROWS_INDEX_DISTANCE: u64 = 10000;
-const PROP_TOTAL_SIZE: &str = "tikv.total_size";
-const PROP_SIZE_INDEX: &str = "tikv.size_index";
-const PROP_RANGE_INDEX: &str = "tikv.range_index";
-pub const DEFAULT_PROP_SIZE_INDEX_DISTANCE: u64 = 4 * 1024 * 1024;
-pub const DEFAULT_PROP_KEYS_INDEX_DISTANCE: u64 = 40 * 1024;
-
-fn get_entry_size(value: &[u8], entry_type: DBEntryType) -> std::result::Result<u64, ()> {
-    match entry_type {
-        DBEntryType::Put => Ok(value.len() as u64),
-        DBEntryType::BlobIndex => match TitanBlobIndex::decode(value) {
-            Ok(index) => Ok(index.blob_size + value.len() as u64),
-            Err(_) => Err(()),
-        },
-        _ => Err(()),
-    }
-}
 
 #[derive(Clone, Debug, Default)]
 pub struct MvccProperties {
```

Comment:
why not move it?

Reply:
It contains the concept of MVCC which should not be put in engine_rocks and it depends on tikv/src/storage.